server/views/applications/pages/further-considerations/room-sharing.njk

```diff
@@ -0,0 +1,61 @@
+{% extends "../layout.njk" %}
+
+{% block questions %}
+  <h1 class="govuk-heading-l">
+    <span class="govuk-caption-l">Room sharing</span>
+    Further placement considerations
+  </h1>
+
+  <p>This information is used to match a person with a suitable room in a suitable premises.</p>
+
+  {% for key, value in page.questions %}
+
+    {% set yesDetails %}
```

Comment:
My (probably wrong) understanding of how these conditionals work is that if you answered 'yes' to any one of these questions the box would pop up for all of the questions but, as I said, I'm probably wrong!

Reply:
Yeah, the `set` is only valid in that part of the loop. I did wonder if that would happen, but it seems to work as expected